src/locales/en.yaml

```diff
@@ -0,0 +1,44 @@
+username: Username
+password: Password
+email: Email
+helpText: Reset Password
+email_me_updates: Email me updates about Webmaker
+join_webmaker: Join Webmaker
+by_joining: "By joining, I agree to Mozilla Webmaker's {linkTerms} and {linkPrivacyPolicy}"
+privacy_policy: Privacy Policy
+terms: Terms
+already_join_signin: Already joined? {signinLink}
+signin: Sign in
+dont_have_account: Don"t have an account? {joinWebmakerLink}
+errorUsernameOrPassword: Looks like there might be a problem with your username or password.
+color_editor: color: {rgbaValue};
+Red: Red
+Green: Green
+Blue: Blue
+Opacity: Opacity
+my_project: My project
+log_out: Log out
+create_a_project: "+ Create a Project"
+open_style_guide: Open Style Guide
+reset_share_preferences: Reset Shared Preferences
+loading: Loading!
+title: Title
+done: Done
+edit_label: Edit Label
+change_link_dest: Change Link Destination
+set_link_dest: Set Link Destination
+corner_radius: Corner Radius
+Font: Font
+background_color: Background Color
+no_project_found: Sorry, no projects found.
+error_featured: Error getting discovery projects
+error_featured_404: No discovery projects found
+error_update_element: There was an error updating the element
+error_element: Error loading element
+error_element_404: No element found
+error_create_make: Tried to create project when no session was found
+error_create_element: There was an error creating an element
+error_delete_element: There was a problem deleting the element
+error_page: There was an error getting the page to load
+error_page_404: Could not find the page to load
+error_404_element_save: Could not find the element to save
```

Comment:
If we're not using this yet, then I don't think it should be in this PR. Do you want to make a new ticket for using YAML?